packages/block-editor/README.md

```diff
@@ -387,6 +387,10 @@ _Type_
 
 -   `Object` 
 
+<a name="transformStyles" href="#transformStyles">#</a> **transformStyles**
+
+Internal dependencies
```

Comment:
This isn't accurate documentation. I think we should use the "Internal dependencies" text only for `import`, whereas we're using it as `export` currently.

I'd suggest:

-  Remove the "Internal dependencies" DocBlock
- Add sufficient documentation for `transformStyles`

Reply:
Suggestion applied 👍